scheduler/src/cook/mesos/api.clj

```diff
@@ -2209,6 +2235,14 @@
                    :handler (read-jobs-handler-multiple conn framework-id is-authorized-fn
                                                         retrieve-sandbox-directory-from-agent)}}))
 
+        (c-api/context
+          "/info" []
+          (c-api/resource
+            {:get {:summary "Returns info about this Cook scheduler instance's setup. No authentication required."
```

Comment:
I don't like the fact that the `No authentication required.` logic is sitting far away in `conditional-auth-bypass`. But not sure if there is a way to get past it.

Reply:
I agree this isn't ideal, but I don't have a good solution. I feel like pulling this out to add another level of indirection wouldn't necessarily make it easier to track down. But I did add a comment above this line that references `cook.components/conditional-auth-bypass`.